examples/megacli.cpp

```diff
@@ -5983,6 +5997,22 @@ static void process_line(char* l)
 
                         return;
                     }
+                    else if (words[0] == "recentnodes")
```

Comment:
New options should be added to the `help` menu, with their params

Reply:
Yes, I agree - will PM you about this